config/kubernetes/org.yaml

```diff
@@ -905,6 +908,7 @@ members:
 - ruiwen-zhao
 - runcom
 - russellb
+- s278gupt
```

Comment:
Maybe this is failing the test. Most probably numbers come after punctuations in the lexical order. TIL if that's true,

Reply:
I was thinking the same. Figured it would throw an error if the sorting was wrong. Fix is on the way

Reply:
Done ✅

Reply:
Cool! It was actually that. New thing learned.

Reply:
I am never going to remember it. It is going to be one of those things where I always do the wrong thing first 😆